bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/TelescopePosTableWidget.java

```diff
@@ -193,8 +183,44 @@ public String formatMagnitude(Magnitude.Band band) {
             }
 
             @Override public Icon getIcon() {
-                if (!isActiveGuideProbe) return errorIcon;
-                else return GuidingIcon.apply(quality.getOrElse(AgsGuideQuality.Unusable$.MODULE$), enabled);
+                return isActiveGuideProbe ?
+                        GuidingIcon.apply(quality.getOrElse(AgsGuideQuality.Unusable$.MODULE$), enabled) :
+                        errorIcon;
+            }
+        }
+
+        static final class BagsTargetRow extends NonBaseTargetRow {
+            final boolean isActiveGuideProbe;
+            final Option<AgsGuideQuality> quality;
+            BagsTargetRow(final boolean isActiveGuideProbe, final Option<AgsGuideQuality> quality, final boolean enabled,
+                          final GuideProbe probe, final SPTarget target, final Option<WorldCoords> baseCoords,
+                          final Option<Long> when) {
+                super(enabled, String.format("%s (BAGS)", probe.getKey()), target, baseCoords, when);
+                this.isActiveGuideProbe = isActiveGuideProbe;
+                this.quality = quality;
+            }
+
+            private static final Icon bagsEnabled  = bagsIcon(true);
+            private static final Icon bagsDisabled = bagsIcon(false);
+            private static final int  bagsGap      = 5;
+
+            private static Icon bagsIcon(boolean enabled) {
+                final BufferedImage img = new BufferedImage(16, 16, BufferedImage.TYPE_INT_ARGB);
```

Comment:
This is sort of a placeholder for now, since we might (I suspect will) change this at some point in the future.
I was using the same size as the AGS feedback icons since they are displayed beside them in the table, which where the size came from, but this is probably something we can avoid worrying about until now until a better solution arises, since the icons are not in the optimal place anyway.